lib/ghtorrent/adapters/mongo_persister.rb

```diff
@@ -55,7 +55,18 @@ def initialize(set)
 
     def store(entity, data = {})
       super
-      mongo[entity].insert_one(data).to_s
+      begin
+        mongo[entity].insert_one(data).to_s
+      rescue StandardError => e
+        # In the interest of best effort, catch errors where the document is too big for Mongo
+        # Not great approach but the exception from Mongo is pretty generic.
+        if e.message.include?("Request size is too large")
```

Comment:
I don't think I 've ever seen this. Is it perhaps a MongoDB misconfiguration on your side? In any case, this is too fragile to include in the code base.